adapters/philips-hue/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * index.js - Loads the Philips Hue bridge API adapter.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+'use strict';
+
+var PhilipsHueAdapter = require('./philips-hue-adapter');
+var rp = require('request-promise-native');
```

Comment:
Thanks for the heads-up! I assumed request-promise-native would have request as a dependency instead of a peer dependency.